staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -187,6 +192,32 @@ func (a *GenericAdmissionWebhook) loadConfiguration(attr admission.Attributes) (
 	return hookConfig, nil
 }
 
+type versioned struct {
```

Comment:
rename `versionedAttributes`, then the concept is clear.

Reply:
Done.